invenio_previewer/assets/semantic-ui/js/invenio_previewer/videojs.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2023 Northwestern University.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import videojs from 'video.js';
+
+// If something needs to be done in the future, add it here.
+// This file is only needed to get the video.js javascript bundled.
+(function () {
+
+})();
```

Comment:
Any more efficient way to include video.js without an empty placeholder file?

Reply:
I am not a webpack expert, but if there is a better way, I'd like to hear it too. My understanding is that we need a js file with the `import videojs from 'video.js;` for webpack to find videojs and bundle it when it follows the import graph. Otherwise we don't have a bundle with videojs we can choose to import in the jinja template file.

Reply:
Not me either, but using an empty file for bundling video.js feels a bit off to me. Not aware of an alternative, but it seems like there should be a cleaner way in webpack.
maybe lazy load it when it's called?

Reply:
Resolved via https://github.com/inveniosoftware/invenio-previewer/pull/194#discussion_r1431029911 :+1: